iceoryx_hoofs/test/moduletests/test_relative_pointer.cpp

```diff
@@ -72,7 +72,7 @@ typedef testing::Types<uint8_t, int8_t, double> Types;
 
 TYPED_TEST_SUITE(RelativePointer_test, Types, );
 
-/// @todo #605 the tests should be reworked in a refactoring of relative pointers
+/// @todo iox-#605 the tests should be reworked in a refactoring of relative pointers
```

Comment:
This issue was already closed so I reopened it again.

Reply:
Thank you!